BedrockCore.cpp

```diff
@@ -135,7 +135,7 @@ BedrockCore::RESULT BedrockCore::peekCommand(unique_ptr<BedrockCommand>& command
             if (!_db.beginTransaction(exclusive ? SQLite::TRANSACTION_TYPE::EXCLUSIVE : SQLite::TRANSACTION_TYPE::SHARED)) {
                 STHROW("501 Failed to begin " + (exclusive ? "exclusive"s : "shared"s) + " transaction");
             }
-            if (exclusive) {
+            if (exclusive && command->writeConsistency != SQLiteNode::QUORUM) {
```

Comment:
Do we also need this check in `processCommand`? (the other place that can change the timeout)

Reply:
Yes, adding.